docs/source/user-guide/nlp/coming_soon.rst

```diff
@@ -1,11 +0,0 @@
-'.. _coming_soon:
```

Comment:
Note for future - had this already been merged into stable etc. then would say to add this path to the redirects in the sphinx conf.py file (see syntax there)
but since it wasn't then no need
(this is mainly for SEO purposes - and also good in general when renaming etc. to make sure we don't have broken links)